.github/workflows/main.yml

```diff
@@ -2,35 +2,49 @@ name: CI
 
 on:
   push:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Do we need to run this daily?

Reply:
We don't need, but we want to. Release branch is unlikely to have a lot of changes, so we want to run the CI periodically, to make sure it's still working.